<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editMenu</class>
 <widget class="QWidget" name="editMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="menuData">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>127</number>
   </attribute>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="itemName"/>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="itemPrice"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveMenu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
